packages/client/components/TeamInvitationSSO.tsx

```diff
@@ -26,12 +27,13 @@ const TeamInvitationSSO = (props: Props) => {
     const loginWithSAML = async () => {
       const invitationToken = localStorage.getItem(LocalStorageKey.INVITATION_TOKEN)!
       submitMutation()
-      const {token, error} = await getTokenFromSSO(ssoURL)
+      const {token, error, ga4Args} = await getTokenFromSSO(ssoURL)
       if (!token) {
         onError(new Error(error || 'Error logging in'))
         return
       }
       atmosphere.setAuthToken(token)
+      emitGA4SignUpEvent(ga4Args!)
```

Comment:
+1 Could you please add a comment that this is ok because it's valid if token is valid?